source/pool/contracts/Pool.sol

```diff
@@ -73,6 +74,7 @@ contract Pool is Ownable {
    */
   constructor(uint256 _scale, uint256 _max) public {
     require(_max <= MAX_PERCENTAGE, "MAX_TOO_HIGH");
+    require(_scale <= MAX_SCALE, "MAX_TOO_HIGH");
```

Comment:
`SCALE_TOO_HIGH`

Reply:
Thank you. I was struggling with `MAX_SCALE_TOO_HIGH` and hated it.